CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 ### Features
 - Added support for `.NET 9` (preview) ([#3699](https://github.com/getsentry/sentry-dotnet/pull/3699))
+- Added support for 16 KB page sizes on Android ([#3723](https://github.com/getsentry/sentry-dotnet/pull/3723))
```

Comment:
We need to reword this though that we're only fixing 1 warning.

Or, wait for the Android release, bump it in this PR too, change the title of the PR (remove `libsentrysupplemental`) and ship it (keep changelog then)